core/src/test/java/hudson/UtilTest.java

```diff
@@ -304,7 +299,7 @@ public void testDeleteFile_onWindows() throws Exception {
                 Util.deleteFile(f);
                 fail("should not have been deletable");
             } catch (IOException x) {
-                assertThat(calcExceptionHierarchy(x), hasItem(c));
+                assertThat(calcExceptionHierarchy(x), hasItem(FileSystemException.class));
```

Comment:
Small cleanup since I was already looking at this test to address the failure.

Reply:
Should ideally have been filed as a separate PR to assess not breaking pre-existing behaviour IMO. Or at least pushed as the first commit alone to trigger the PR build and see a green :white_check_mark:

Reply:
Reverted locally `src/main/java` to `master` content, and the `UtilTest` still succeeds, so :+1:.

Reply:
Good point, I'll make sure to make test changes in a separate PR or push them as a first commit in the future.